components/Footer.js

```diff
@@ -1,12 +1,58 @@
 export default () => (
-    <div>
-        <script 
-            src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0/jquery.min.js" 
-            integrity="sha256-JmvOoLtYsmqlsWxa7mDSLMwa6dZ9rrIdtrrVYRnDRH0=" 
-            crossOrigin="anonymous"></script>
-        <script 
-            src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" 
-            integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" 
-            crossOrigin="anonymous"></script>
-    </div>
-)
\ No newline at end of file
+  <div
+    style={{
+      paddingTop: "15px",
+      paddingBottom: "15px",
+      position: "absolute",
+      bottom: 0,
+      width: "100%"
+    }}
+    className="bg-primary"
+  >
+    <footer className="">
+      <div className="text-center">
+        <div style={{fontSize: "16px"}}>
+          Made with <span style={{ color: "#ad1010" }}>❤</span> for
```

Comment:
Better if we use the HTML symbol entities. You can check this one https://www.w3schools.com/charsets/tryit.asp?deci=9829&ent=hearts.